public/js/auth.js

```diff
@@ -0,0 +1,7 @@
+function sign_up_user() {
```

Comment:
Identifier 'sign_up_user' is not in camel case.<br>'sign_up_user' is defined but never used.